bin/client/Cargo.toml

```diff
@@ -22,6 +22,8 @@ spin.workspace = true
 async-trait.workspace = true
 tracing.workspace = true
 revm.workspace = true
+serde = { version = "1.0.208", default-features = false, features = ["derive"] }
+serde_json = { version = "1.0.125", default-features = false }
```

Comment:
oof can we feature flag this? Will bloat the client no?

Reply:
hmm, we can, but it's actually not too too bad. Release binary for `kona` is only `2.9MB`, vs. `op-program`'s 35mb+